crates/fuel-core/src/service/adapters/txpool.rs

```diff
@@ -102,9 +118,48 @@ impl fuel_core_txpool::ports::PeerToPeer for P2PAdapter {
             Ok(())
         }
     }
+
+    async fn request_tx_ids(
+        &self,
+        peer_id: Vec<u8>,
+    ) -> anyhow::Result<Vec<fuel_core_txpool::types::TxId>> {
```

Comment:
maybe you want to import TxId at the top given you're using the fully qualified type in two places

Reply:
Done :)